src/screens/settings/color_settings.html

```diff
@@ -0,0 +1,174 @@
+        <div class="color-mode-display">
+            <section class="color-preview">
+                <h3>Preview</h3>
+ <!-- ..............START OF PREVIEW-DISPLAY  -->
+                <div class="preview-display">
+                    <p>Wow! Nice theme!</p>
+                    <div class="preview-display1">
+                        <p>Hi guys!</p>
+                    </div>
+                    <div class="preview-display-right">
+                        <p>This is my favourite color</p>
+                    </div>
+                </div>
+ <!-- ..............END OF PREVIEW-DISPLAY  -->
+
+                <p>COLOR</p>
+ <!-- ..............START OF COLOR-PICK  -->
+                <div class="color-pick">
+                    <div class="color-pick-item1">
+                        <div 
+                            class="color-pick1-item1">
+                            <svg 
+                            width="25" 
+                            height="25" 
+                            viewBox="0 0 25 25" 
+                            fill="none" 
+                            xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+                                <rect 
+                                y="25" 
+                                width="25" 
+                                height="25" 
+                                rx="12.5" 
+                                transform="rotate(-90 0 25)" fill="#150AA1"/>
+                                <rect 
+                                x="8" 
+                                y="8" 
+                                width="10" 
+                                height="10" 
+                                fill="url(#pattern0)"/>
+                                <defs>
+                                <pattern id="pattern0" patternContentUnits="objectBoundingBox" width="1" 
+                                height="1">
+                                <use 
+                                xlink:href="#image0_270_165" transform="scale(0.0111111)"/>
+                                </pattern>
+                                <image 
+                                id="image0_270_165" 
+                                width="90" 
+                                height="90" 
+                                xlink:href="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAFoAAABaCAYAAAA4qEECAAAACXBIWXMAAAsTAAALEwEAmpwYAAABa0lEQVR4nO3ZsUoDQRRG4TWF0cLnthELu3Q+g0XYJ7ITRCKWRxZXECxMIvPP7M75IBDYIvcehrDZDIMkSZIkSZIkSZIUAWyBO+AFOAA74Drz6X1F3vPbrvZsPUSevNWer4fIk9faM/YQeXJfe84eIk/Xt7VnXSzgEnj6I/IIXNWedbGMHGDkACMHGDnAyAFGDjBygJEDjBxg5AAjBxg5wMgBRg4wcoCRA4wcYOSAxUcGNsDF0DCW/EcqcAM8Ah/AO/DQ4qAsOfJkjtz0wKwg8mY+yc0OztIj/wh9aHUB1hD52/yd3NwirClyqwvR4EyruzeloVlWuyANzBBRc1F6iVxzYXqLXGNxeo2cDEDvkRMhMHL5IBi5fBiMXD4QRj7Of0Jh5NOcEwwjn+eUcBg59oRtv7qncGlHnta+f4w0EHs0cvnYo5HLn+zRyOVjj0YuH3s0cplbv1vgeX5N772FkyRJkiRJkiRJ0vDlEzK9oz9IQJAwAAAAAElFTkSuQmCC"/>
```

Comment:
You have to put this svg in to its own file like "name.svg". It appears to be too long for this space here